arch/RegisterDefnsJSON.cpp

```diff
@@ -85,13 +85,11 @@ namespace atlas
             constexpr sparta::RegisterBase::Definition::HintsT hints = 0;
             constexpr sparta::RegisterBase::Definition::RegDomainT regdomain = 0;
 
-            const bool writable = id != 0 || group_num != 0;
-
             sparta::RegisterBase::Definition defn = {
                 id,        name,          group_num,     group,           group_idx,
                 desc,      bytes,         field_defns,   bank_membership, aliases,
                 subset_of, subset_offset, initial_value, hints,           regdomain,
-                writable};
+                true};
```

Comment:
This was changed to allow x0 to be "written."  In the background, `x0` is masked off, meaning a write to it is always ignored.